sympy/physics/mechanics/actuator.py

```diff
@@ -1023,8 +1023,7 @@ class CoulombKineticFriction(ForceActuator):
     - The actuator assumes relative motion is non-zero.
     - The normal force is assumed to be a non-negative scalar.
     - The resultant friction force is opposite to the velocity direction.
-    - Each point in the pathway is fixed in separate objects that are sliding relative to each other.
-    These two points represent contact points on the two sliding objects.
+    - Each point in the pathway is fixed in separate objects that are sliding relative to each other. These two points represent contact points on the two sliding objects.
```

Comment:
They don't technically have to represent the contact points, but they do need to be fixed in the mutually sliding objects.